R/fileValidation.r

```diff
@@ -189,18 +189,18 @@ file_separator <- function(data, meta) {
     if ((grepl(",", data) == FALSE) && (grepl(",", meta) == FALSE)) {
       output <- list(
         "result" = "FAIL",
-        "message" = paste0("Neither of the data and metadata files appear to be using a comma delimited structure. <br> - See the ", "<a href='https://rsconnect/rsc/stats-production-guidance/ud.html#data_format' target='_blank'>guidance on how to save the files</a>", " if you are unsure.")
+        "message" = paste0("Neither of the data and metadata files appear to be using a comma delimited structure. <br> - See the ", "<a href='https://dfe-analytical-services.github.io/analysts-guide/statistics-production/ud.html#data_format' target='_blank'>guidance on how to save the files</a>", " if you are unsure.")
```

Comment:
I don't know whether this is a quirk of the browser I'm using but this link takes me to the top of the Open Data Standards page and not the relevant section - not the end of the world but slightly irritating

Reply:
Ah damn, this is my laziness coming through - the anchor tag has changed to use kebab case in quarto so should be `#data-format`, I'll update and double check the others too

Reply:
Fixed this in the latest commit

Reply:
I got caught out by this yesterday too!